### ISSUE TITLE
Razer viper ultimate not responding

### ISSUE THREAD

**Issue by joeyiscool12** [1kc/razer-macos#252]
Hi, I have a razer viper ultimate, however, it does not respond when I try to change the RGB with the software. I have tried uninstalling and reinstalling the software, restarting my computer, switching memory profiles, also I forgot to mention I have the viper ultimate with a charging dock which does change RGB when I adjust it in the software. Which therefore means it is not my computer's issue and rather an error in the support. The razer viper ultimate is confirmed supported, and it is disappointing to see that it does not work when I try to adjust RGB. I sincerely apologize for any possible inconvenience I may have caused, hope this issue can be fixed soon, Thanks!

**Comment by 1kc** [comment]
One thing you could try is to download one of the older versions. It could have been broken in the new release.

**Comment by joeyiscool12** [comment]
@1kc  hi, no I tried downloading several older versions but it still didn't work, do you have any other suggestions?

**Comment by 1kc** [comment]
That’s a bummer, has anyone had it working ever?

**Comment by Ethirallan** [comment]
I searched through the issues and found the PR for adding support. With working dock support it was released in [v0.2.4](https://github.com/1kc/razer-macos/releases/tag/v0.2.4). This version is working without any problems so far.

**Comment by joeyiscool12** [comment]
> I searched through the issues and found the PR for adding support. With working dock support it was released in [v0.2.4](https://github.com/1kc/razer-macos/releases/tag/v0.2.4). This version is working without any problems so far.

I see, thanks!

**Comment by ddavini** [comment]
> I searched through the issues and found the PR for adding support. With working dock support it was released in [v0.2.4](https://github.com/1kc/razer-macos/releases/tag/v0.2.4). This version is working without any problems so far.

Nope. Still doesn't work for me. The dock works okay, but not the mouse. I'm on M1 running 12.3, if that's help somehow.

I'll try to look into the code if I have some time.

**Comment by ddavini** [comment]
> > I searched through the issues and found the PR for adding support. With working dock support it was released in [v0.2.4](https://github.com/1kc/razer-macos/releases/tag/v0.2.4). This version is working without any problems so far.
> 
> Nope. Still doesn't work for me. The dock works okay, but not the mouse. I'm on M1 running 12.3, if that helps somehow.
> 
> I'll try to look into the code if I have some time.

LOL. I'm dumb. The gd brightness was zero.
<img width="493" alt="Screen Shot 2022-04-01 at 1 10 07 PM" src="https://user-images.githubusercontent.com/13496194/161252794-e6ffcb2f-61ad-42c6-af9f-6216c79e2752.png">
